### DISCUSSION TITLE
Force app to use GL backend

### DISCUSSION THREAD

**Discussion by drtheodor** [slint-ui/slint#1196] [Q&A] [answered]
Is there a way I can force Slint to use GL backend(instead of including both - qt and gl include only gl, to make binary size smaller)?

**Comment by ogoffart** [comment, marked as answer]
With features flags. In Cargo.toml it would look like

```
slint = { version="0.2", default-features=false, features = ["backend-gl-all", "compat-0-2-0"]}
```
